eclass/multiprocessing.eclass

```diff
@@ -64,17 +64,36 @@ get_nproc() {
 	fi
 }
 
+# @FUNCTION: _get_all_makeopts
+# @INTERNAL
+# @DESCRIPTION:
+# Returns ${MAKEOPTS} ${GNUMAKEFLAGS} ${MAKEFLAGS}.
+_get_all_makeopts() {
+	echo "${MAKEOPTS} ${GNUMAKEFLAGS} ${MAKEFLAGS}"
+}
+
+# @FUNCTION: get_makeopts_jobs
+# @USAGE: [default-jobs]
+# @DESCRIPTION:
+# Return the number of jobs extracted from the make options (MAKEOPTS,
+# GNUMAKEFLAGS, MAKEFLAGS). If the make options do not specify a number,
+# then either the provided default is returned, or 1.
+get_makeopts_jobs() {
+	local makeopts="$(_get_all_makeopts)"
```

Comment:
Why do you need a helper variable? I'm not saying it is wrong but I suppose the logic would be easier to follow (as in people wouldn't wonder why the extra var) if the call was inline.

Reply:
Thanks for your feedback. You are right, the helper variable is not strictly required. Personally, however, I find it more readable with the helper variable, but ymmv. On the other hand, this is not a hill for me to die on.

Reply:
It was more readable with the first version of the patch. But you are right, it is probably not required in the current version.